package/src/views/useTouchHandler.ts

```diff
@@ -56,8 +53,8 @@ const useInternalTouchHandler = (
 
         const extendedTouchInfo: ExtendedTouchInfo = {
           ...touch,
-          velocityX: prevVelocityRef.current[touch.id]?.x,
-          velocityY: prevVelocityRef.current[touch.id]?.y,
+          velocityX: prevVelocityRef.current[touch.id]?.x ?? 0,
+          velocityY: prevVelocityRef.current[touch.id]?.y ?? 0,
```

Comment:
Perfect :) Thanks for finding this and providing a fix!

Reply:
There might be a small clean-up there.
1. The velocity is computed above and might be needed to be set to 0 if `timeDiffseconds === 0`. If should be done at the same place.
2. The velocity is computed and stored in x and y property of the previous touch, could it be velocityX and velocityY? Maybe it shouldn't be stored in the ref at all? It looks like we are calculating the velocity based on a velocity in the next frame (might be wrong).

Reply:
1: So something like:

```
if (timeDiffseconds > 0) {
  prevVelocityRef.current[touch.id] = {
    x: distX / timeDiffseconds / PixelRatio.get(),
    y: distY / timeDiffseconds / PixelRatio.get(),
  };
} else {
  prevVelocityRef.current[touch.id] = { x: 0, y: 0 }; // <- new
}
```
But wouldn't this be covered by the fix proposed in this PR?

2: We're using a ref to store the previous one so that we only calculate velocities when we move - on end/cancel we should use the previously calculated value - since end/cancel are called immediately (timeDiffSeconds === 0) after the last active.

Reply:
1. Exactly and this is part of this PR right? Since the goal is to have this value always defined and the types to be able to statically catch such "bug"?
2. Got it and it makes sense.

Reply:
Yes and yes :) So then we're good to go with this one?

Reply:
I understand. Thanks for your review.